src/_pytest/fixtures.py

```diff
@@ -424,7 +424,7 @@ def module(self):
         return self._pyfuncitem.getparent(_pytest.python.Module).obj
 
     @scopeproperty()
-    def fspath(self):
+    def fspath(self) -> py.path.local:
```

Comment:
Note that `mypy-diff` (https://github.com/pytest-dev/pytest/pull/6486) would complain here for this to be missing still (since mypy does not know about py.path.local).